tests/test_getters.py

```diff
@@ -47,24 +48,37 @@ class TestGetters(asynctest.TestCase):
     async def test_get_image(self):
         day_obs = 20200315
         seq_num = 139
-        data_id = {"day_obs": day_obs, "seq_num": seq_num, "detector": 0, "instrument": 'LATISS'}
+        data_id = {
+            "day_obs": day_obs,
+            "seq_num": seq_num,
+            "detector": 0,
+            "instrument": "LATISS",
+        }
+
+        datapath = "/readonly/repo/main/"  # FIXME
+        best_effort_isr = BestEffortIsr(datapath, repodirIsGen3=True)
```

Comment:
As with the other PR, I think this should be switched to calling the constructor with a butler, rather than a path.